src/utils/prompts.py

```diff
@@ -29,3 +59,50 @@
 * q3
 
 """
+        return ret
+
+
+class RedditPromptGenerator(PromptGenerator):
+    @property
+    def source(self) -> str:
+        return "reddit"
+
+    def get_summarization_template(self, method: PromptMethod) -> str:
+        if method == PromptMethod.stuff:
+            ret = """\
+Given the following reddit submissions
+
+{text}
+
+I want you to provide a short summary and produce three questions that cover the discussed topics.
```

Comment:
How does the LLM know if the post has a high number of upvotes or if the redditor has a high karma?
I think it's interesting to include this information but I'm not sure the LLMs uses it (the text only is used here).
Unless I'm missing something?